docs/concepts/Auditing-Packages.md

```diff
@@ -16,24 +16,107 @@ This involves identifying vulnerabilities, evaluating risks, and making recommen
+<ItemGroup>
+    <NuGetAuditSuppress Include="https://github.com/advisories/XXXX" />
+</ItemGroup>
+```
+
+Similar to the other NuGet audit configuration properties, `NuGetAuditSuppress` items can be defined at the project or repository level.
+
+`NuGetAuditSuppress` is available for PackageReference projects starting from NuGet 6.11, Visual Studio 17.11, and the .NET 8.0.400 SDK.
+It is available for packages.config with Visual Studio 17.12 and NuGet 6.12.
+
+### Warning codes
+
+| Warning Code | Reason |
+|--------------|----------|
+| [NU1900](../reference/errors-and-warnings/NU1900.md) | Error communicating with package source, while getting vulnerability information. |
+| [NU1901](../reference/errors-and-warnings/NU1901-NU1904.md) | Package with low severity detected |
+| [NU1902](../reference/errors-and-warnings/NU1901-NU1904.md) | Package with moderate severity detected |
+| [NU1903](../reference/errors-and-warnings/NU1901-NU1904.md) | Package with high severity detected |
+| [NU1904](../reference/errors-and-warnings/NU1901-NU1904.md) | Package with critical severity detected |
+| [NU1905](../reference/errors-and-warnings/NU1905.md) | An audit source does not provide a vulnerability database |
+
+You can customize your build to treat these warnings as errors to [treat warnings as errors, or treat warnings not as errors](/dotnet/csharp/language-reference/compiler-options/errors-warnings#warningsaserrors-and-warningsnotaserrors).
+For example, if you're already using `<TreatWarningsAsErrors>` to treat all (C#, NuGet, MSBuild, etc) warnings as errors, you can use `<WarningsNotAsErrors>NU1901;NU1902;NU1903;NU1904</WarningsNotAsErrors>` to prevent vulnerabilities discovered in the future from breaking your build.
+Alternatively, if you want to keep low and moderate vulnerabilities as warnings, but treat high and critical vulnerabilities as errors, and you're not using `TreatWarningsAsErrors`, you can use `<WarningsAsErrors>NU1903;NU1904</WarningsAsErrors>`.
+
+> [!NOTE]
+> MSBuild properties for message severity such as `NoWarn` and `TreatWarningsAsErrors` are not supported for packages.config projects.
+
+## `dotnet list package --vulnerable`
+
+Once a project is successfully restored, [`dotnet list package`](/dotnet/core/tools/dotnet-list-package) has a `--vulnerable` argument to filter the packages based on which packages have known vulnerabilities.
+Note that `--include-transitive` is not default, so should be included 
+
+## Actions when packages with known vulnerabilities are reported
+
+We also have a [blog post](https://devblogs.microsoft.com/nuget/nugetaudit-2-0-elevating-security-and-trust-in-package-management/) which discusses our recommended method for taking action when a package with a known vulnerability is found to be used by your project, and tools to help get more information.
+
 ### Security vulnerabilities found with updates
 
 If security vulnerabilities are found and updates are available for the package, you can either:
```

Comment:
What do other people think about this section of the docs (and the next 50 unmodified lines)?

It feels out of place to me. Maybe we should be doing more of it in our docs, but our docs are mostly an unopinionated reference to our features, whereas this is providing opinions on what to do.

Now that we have the blog post out, and I've linked to it in this page, should we remove this section from these docs?

I was also thinking that instead of having a single page on audit, it could be a section in the table of contents:

```
Auditing packages
+ Overview
+ Auditing during restore
+ Auditing with dotnet list package
```

And if we want to go down the route of having opinionated recomendations, there could also be "Recommendations for package consumers" which could contain the information I'm proposing to remove here (but also be a duplicate of the blog post).

Any comments?

Reply:
will consider for a different PR